src/global/const.h

```diff
@@ -207,10 +213,12 @@
 #define LOW_LIGHT 0x1400 // = 5120
 #define HIGH_LIGHT 0x1000 // = 4096
 
-#define VIEW_NEAR 127
+#define VIEW_NEAR 127 / ANIM_SCALE
 #define VIEW_FAR (0x14 * 0x400)
-#define DEPTH_Q_END (20 * 1024) // = 20480
-#define DEPTH_Q_START ((DEPTH_Q_END) - (8 * 1024)) // = 12288
+//#define DEPTH_Q_END (20 * 1024)
+#define DEPTH_Q_END (40 * 1024)
+//#define DEPTH_Q_START ((DEPTH_Q_END) - (8 * 1024))
+#define DEPTH_Q_START ((DEPTH_Q_END - 1024)) // - (16 * 1024))
```

Comment:
Why?

Reply:
fixes object clipping near the camera I believe